openapi.yaml

```diff
@@ -1,279 +1,662 @@
+          schema:
+            type: string
+        - in: query
+          name: filter[latitude]
+          description: "Latitude of the location. Should be used with lon parameter"
+          required: false
+          schema:
+            type: number
+            format: float
+        - in: query
+          name: filter[longitude]
+          description: "Longitude of the location. Should be used with lat parameter"
+          required: false
+          schema:
+            type: number
+            format: float
+        - in: query
+          name: filter[distance]
+          description: "The radius of the circle centred on the specified location. Default: 2"
+          required: false
+          schema:
+            type: number
+            format: float
+        - in: query
+          name: filter[distanceUnit]
+          description: "Distance can be specified in following units:
+            Mile - mi or miles (default);
+            Yard - yd or yards;
+            Feet - ft or feet;
+            Inch - in or inch;
+            Kilometer - km or kilometers;
+            Meter - m or meters;
+            Centimeter - cm or centimeters;
+            Millimeter - mm or millimeters;
+            Nautical mile - NM, nmi or nauticalmiles
+          "
+          required: false
+          schema:
+            type: string
+            enum: 
+              - mi
+              - miles
+              - yd
+              - yards
+              - ft
+              - feet
+              - in
+              - inch
+              - km
+              - kilometers
+              - m
+              - meters
+              - cm
+              - centimeters
+              - mm
+              - millimeters
+              - nmi
+              - NM
+              - nauticalmiles
+        - in: query
+          name: filter[isOpen]
+          description: "Set to true to only search for open restaurants/buildings. Default: false to search for all the results"
+          required: false
+          schema:
+            type: boolean
+        - in: query
+          name: filter[campus][oneOf]
+          description: "Name of the campus. Valid options are: corvallis, extension, cascades, hmsc, and other."
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - corvallis
+                - extension
+                - cascades
+                - hmsc
+                - other
+        - in: query
+          name: filter[type]
+          description: "Type of location. Valid options are: building, dining, cultural-center, parking, and other."
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - building
+                - dining
+                - cultural-center
+                - parking
+                - other
+        - in: query
+          name: filter[giRestroom]
```

Comment:
Just my two cents, I think `filter[hasGiRestroom]` is easier to understand.

Reply:
Yeah, I would agree